python/paddle/v2/fluid/tests/test_recv_op.py

```diff
@@ -28,6 +29,7 @@ def test_send(self):
         p = Process(target=self.init_serv, args=(place, ))
         p.daemon = True
         p.start()
+        time.sleep(5)
```

Comment:
Maybe change 5s to something lower (1s?). 5s per unit test is a lot of time considering how much unit tests we run everyday.